pkg/oc/cli/status/status.go

```diff
@@ -74,31 +74,29 @@ func NewStatusOptions(streams genericclioptions.IOStreams) *StatusOptions {
 // NewCmdStatus implements the OpenShift cli status command.
 // baseCLIName is the path from root cmd to the parent of this cmd.
 func NewCmdStatus(name, baseCLIName, fullName string, f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Command {
-	opts := NewStatusOptions(streams)
-
+	o := NewStatusOptions(streams)
 	cmd := &cobra.Command{
 		Use:     fmt.Sprintf("%s [-o dot | -s ]", StatusRecommendedName),
```

Comment:
The display of the short flag `-s` in the usage is confusing:
```
$ oc status -s
Error: flag needs an argument: 's' in -s
$ oc status -h
Usage:
  oc status [-o dot | -s ] [flags]
```
If `-s` stands for the global flag `--server`, it is unnecessary to be displayed under `oc status -h`
```
$ oc options
  -s, --server='': The address and port of the Kubernetes API server
```

Reply:
There's not shorthand `-s` thanks for catching that up, should be `--suggest`.